src/module_header.h

```diff
@@ -39,7 +39,6 @@
 #include <stddef.h>
 #include <stdarg.h> /* va_list */
 #include <errno.h>
-#include <string.h> /* strerror(3) */
```

Comment:
It leads to the following build problems:

* https://github.com/tarantool/kafka/issues/89
* https://github.com/tarantool/tuple-keydef/issues/24

Sure, the patch itself is correct. Well, it should be covered by the integration testing.

However I would be more conservative in such changes in a future, because build problems on the user side (and a need to update the modules) doesn't worth it.